www/_posts/2017-11-10-plugins-release.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Plugins Release"
+categories: news
+tags: release plugins
+---
+
+The following plugins were updated today because they are being deprecated:
+
+* cordova-plugin-battery-status@1.2.5
+* cordova-plugin-camera@3.0.0
+* cordova-plugin-contacts@3.0.0
+* cordova-plugin-device@1.1.7
+* cordova-plugin-dialogs@1.3.4
+* cordova-plugin-file-transfer@1.7.0
+* cordova-plugin-file@5.0.0
+* cordova-plugin-geolocation@3.0.0
+* cordova-plugin-globalization@1.0.8
+* cordova-plugin-inappbrowser@1.7.2
+* cordova-plugin-media@4.0.0
+* cordova-plugin-media-capture@2.0.0
+* cordova-plugin-network-information@1.3.4
+* cordova-plugin-splashscreen@4.1.0
+* cordova-plugin-statusbar@2.3.0
+* cordova-plugin-screen-orientation@2.0.2
+* cordova-plugin-vibration@2.1.6
+* cordova-plugin-whitelist@1.3.3
+* cordova-plugin-wkwebview-engine@1.1.4
+* cordova-plugin-test-framework@1.1.6
+
+
+In our [last plugins release](https://cordova.apache.org/news/2017/09/22/plugins-release.html), we deprecated `cordova-plugin-compat` since it got integrated into `cordova-android@6.3.0`. So for this release cycle, we have removed the dependency from plugins that were relying on it and gave the plugins a major version jump. The follow plugins have dropped `cordova-plugin-compat`, `cordova-plugin-camera`, `cordova-plugin-contacts`, `cordova-plugin-file`, `cordova-plugin-geolocation`, `cordova-plugin-media`, and `cordova-plugin-media-capture`.
+
+We have also changed how usage descriptions work in the following plugins: `cordova-plugin-camera`, `cordova-plugin-contacts`, `cordova-plugin-media`, and `cordova-plugin-media-capture`. Usage descriptions are required for **iOS** applications accessing certain apis. Apple wants to know why your app needs certain permissions. We now recommend you add the usage description to your app via `edit-config` tag. View the `iOS Quirks` section of the plugin documentation to see an example of how to use it. [Here](https://github.com/apache/cordova-plugin-camera#ios-quirks) is the example for `cordova-plugin-camera`.
```

Comment:
`cordova-plugin-geolocation` also changed. Maybe we should make clear that this is mandatory or apps will crash